### PR TITLE
Physical Checkout fields visibility changed

### LINKED ISSUE #72: Show shipping fields by default
Because instapage is not responsive and we generally show stuff below the checkout on mobile, let's make the fields show by default. Those fields can be disabled by default and enabled when the checkbox is clicked. When the fields are hovered they will need the **cursor: not-allowed !important;**

I chatted with Miguel and Tristan about this, and they think this is the best route to move forward with it.

Also, let's bold this label: "**Check this box if your shipping address is different from your billing address**"

### PR DESCRIPTION
**Pull request by gaurav-bains**
Physical Checkout fields are always visible now, closes #72

### CODE REVIEW

File: checkout/components/checkoutForm.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ app.component('checkout-form',{
       <div v-if="physicalCheckout" class="physicalCheckoutDiv">
         <h3>Shipping Address:</h3>
         <input type="checkbox" v-model="shippingToggle">&nbsp;Check this box if your shipping address is different from your billing address
```
Comment (pjdarch) [inline checkout/components/checkoutForm.js:18]: Label needs to be bold as per #72

File: dist/checkout/checkout.js [hunk 2]
```diff
@@ -421,7 +419,7 @@ app.component('checkout-form',{
       <div v-if="physicalCheckout" class="physicalCheckoutDiv">
         <h3>Shipping Address:</h3>
         <input type="checkbox" v-model="shippingToggle">&nbsp;Check this box if your shipping address is different from your billing address
```
Comment (pjdarch) [inline dist/checkout/checkout.js:421]: Label needs to be bold as per #72

**Comment by pjdarch** [comment]
Looks okay, but I know of a random trick here.. Not sure if it'll work, but you can disable multiple controls by wrapping them in a `<fieldset>` tag. https://www.w3schools.com/tags/att_fieldset_disabled.asp This would prevent the need from having to set them on multiple inputs.

File: checkout/components/checkoutForm.js (hunk 1 above)
Comment (gaurav-bains) [reply checkout/components/checkoutForm.js]: Thanks @pjdarch 
This is fixed now

File: dist/checkout/checkout.js (hunk 2 above)
Comment (gaurav-bains) [reply dist/checkout/checkout.js]: Thanks @pjdarch
This is fixed now

**Review by pjdarch — APPROVED** [review]

**Pull request merged**